src/app/zap-templates/templates/chip/python-CHIPClusters-py.zapt

```diff
@@ -26,7 +26,7 @@ class ChipClusters:
 {{#chip_server_cluster_commands}}
                 "{{asCamelCased name false}}": {
 {{#chip_server_cluster_command_arguments}}
-                    "{{asCamelCased label}}": "{{asPythonType chipType}}",
+                    "{{asCamelCased label}}": "{{#if (isCharString type)}}str{{else}}{{asPythonType chipType}}{{/if}}",
```

Comment:
Would it be possible to fix rather `asPythonType` function?

Reply:
#5431 methoned the issue that CHAR_STRING repr in CHIP cluster does not match spec, by fix that, we are required to use another type of CHAR_STRING instead of ByteSpan, then we can fix this issue (or have a better way to deal with it).